commit b1d840a4a23bd586059d6654fb60a42114ae7b92
Author: Jamie Strandboge <jdstrand@ubuntu.com>
Date:   Sun Sep 19 01:09:36 2021 -0500

    src/ufw: update copyright year

diff --git a/src/ufw b/src/ufw
index 115d294..7d72395 100755
--- a/src/ufw
+++ b/src/ufw
@@ -2,7 +2,7 @@
 #
 # ufw: front-end for Linux firewalling (cli)
 #
-# Copyright 2008-2018 Canonical Ltd.
+# Copyright 2008-2021 Canonical Ltd.
 #
 #    This program is free software: you can redistribute it and/or modify
 #    it under the terms of the GNU General Public License version 3,
@@ -108,7 +108,7 @@ if __name__ == "__main__":
         sys.exit(0)
     elif pr.action == "version" or pr.action == "--version":
         msg(ufw.common.programName + " " + version)
-        msg("Copyright 2008-2018 Canonical Ltd.")
+        msg("Copyright 2008-2021 Canonical Ltd.")
         sys.exit(0)
 
     try:
