Origin: dce24d1b3a17c819100021d29d9128f0fee2866c
Description: allow check-requirements to pass with later py3
Index: ufw-0.36/tests/check-requirements
===================================================================
--- ufw-0.36.orig/tests/check-requirements
+++ ufw-0.36/tests/check-requirements
@@ -50,16 +50,16 @@ runcmd() {
 # check python
 found_python="no"
 echo -n "Has python: "
-for exe in python2.7 python2.6 python2.5 python3.2 python3.3 python3.4 python3.5 python python3 ; do
+for exe in python2.7 python2.6 python2.5 python3.2 python3.3 python3.4 python3.5 python3.6 python3.7 python python3 ; do
     if ! which $exe >/dev/null 2>&1; then
         continue
     fi
     v=`$exe --version 2>&1 | cut -f 2 -d ' '`
-    if echo "$v" | grep -q "^2.[567]"; then
+    if echo "$v" | grep -q "^2.[5-7]"; then
         echo "pass (binary: $exe, version: $v, py2)"
         found_python="yes"
         break
-    elif echo "$v" | grep -q "^3.[234]"; then
+    elif echo "$v" | grep -q "^3.[2-7]"; then
         echo "pass (binary: $exe, version: $v, py3)"
         found_python="yes"
         break
