Author: Matthias Klose <doko@debian.org>
Description: Fix python version check for Python >= 3.9
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=975912
Forwarded: yes
--- a/tests/check-requirements
+++ b/tests/check-requirements
@@ -59,7 +59,7 @@ for exe in python3 python2 python ; do
         echo "pass (binary: $exe, version: $v, py2)"
         found_python="yes"
         break
-    elif echo "$v" | grep -q "^3.[2-8]"; then
+    elif echo "$v" | grep -q "^3.[1-9][0-9]*"; then
         echo "pass (binary: $exe, version: $v, py3)"
         found_python="yes"
         break
