commit 7054614f0cc0f44469f3cf808b5c69b8958588e7
Author: Jamie Strandboge <jamie@ubuntu.com>
Date:   Sun Feb 16 08:50:09 2020 -0600

    tests/check-requirements: simplify and support python 3.8

diff --git a/tests/check-requirements b/tests/check-requirements
index 0062335..0794d47 100755
--- a/tests/check-requirements
+++ b/tests/check-requirements
@@ -50,7 +50,7 @@ runcmd() {
 # check python
 found_python="no"
 echo -n "Has python: "
-for exe in python2.7 python2.6 python2.5 python3.2 python3.3 python3.4 python3.5 python3.6 python3.7 python python3 ; do
+for exe in python3 python2 python ; do
     if ! which $exe >/dev/null 2>&1; then
         continue
     fi
@@ -59,7 +59,7 @@ for exe in python2.7 python2.6 python2.5 python3.2 python3.3 python3.4 python3.5
         echo "pass (binary: $exe, version: $v, py2)"
         found_python="yes"
         break
-    elif echo "$v" | grep -q "^3.[2-7]"; then
+    elif echo "$v" | grep -q "^3.[2-8]"; then
         echo "pass (binary: $exe, version: $v, py3)"
         found_python="yes"
         break
